### ISSUE TITLE
Issue with Excel Formula After copying ExcelRangeBase

### ISSUE THREAD

**Issue by anujbastola** [EPPlusSoftware/EPPlus#1332]
I have a code that is copying a range of cells to another range with the following code. 

`originalSheet.sheet.Cells[originalSheet.Dimensions].Copy(newWorkSheet.sheet.Cells[beginRowIndex, 1]);`

While debugging the result, I saw that it was adding `isc` in the Formula field on the new worksheet.

I added this code to debug.

```
                for (int row = originalSheet.sheet.Dimension.Start.Row; row <= originalSheet.sheet.Dimension.End.Row; row++)
                {
                    for (int col = originalSheet.sheet.Dimension.Start.Column; col <= originalSheet.sheet.Dimension.End.Column; col++)
                    {
                        var cell = originalSheet.sheet.Cells[row, col];
                        if (cell.Formula != "")
                        {
                            Debug.WriteLine($"Cell {cell.Address} has formula: {cell.Formula}");
                        }
                    }
                }

                originalSheet.sheet.Cells[originalSheet.Dimensions].Copy(newWorkSheet.sheet.Cells[beginRowIndex, 1]);

                Debug.WriteLine("---------------- After Copy--------------------");

                // For loop just for debugging to see what we have in Formula (new copied worksheet)
                for (int row = newWorkSheet.sheet.Dimension.Start.Row; row <= newWorkSheet.sheet.Dimension.End.Row; row++)
                {
                    for (int col = newWorkSheet.sheet.Dimension.Start.Column; col <= newWorkSheet.sheet.Dimension.End.Column; col++)
                    {
                        var cell = newWorkSheet.sheet.Cells[row, col];
                        if (cell.Formula != "")
                        {
                            Debug.WriteLine($"Cell {cell.Address} has formula: {cell.Formula}");
                        }
                    }
                }
```

With the code, I found out that it was adding `**isc**` in the Formula which make it invalid in Excel.

![image](https://github.com/EPPlusSoftware/EPPlus/assets/22779571/31f01769-cea5-4214-99a5-5298c71df42d)

Is this an issue with EPPlus or am I missing something in my code?

**Comment by swmal** [comment]
Thanks for reporting this. `isc` is the textual representation of the intersect operator in EPPlus. I will try to replicate the issue and provide a fix.

**Comment by swmal** [comment]
This fix was released in v7.0.10
